cfg_samples/kanata.kbd

```diff
@@ -149,6 +149,24 @@ If you need help, please feel welcome to ask in the GitHub discussions.
   ;;   linux-output-device-bus-type USB
   ;;   linux-output-device-bus-type I8042
 
+  ;; On Linux, it is possible to configure a debounce algorithm.
+  ;; This is useful in the case of keyboard 'chatter' or 'double taps' due to faulty hardware.
+  ;; The default algorithm is "asym_eager_defer_pk".
+  ;;
+  ;; Example: Set debounce duration to 50ms.
+  ;;   linux-debounce-duration 50
```

Comment:
I think theoretically this implementation should be very feasible in Windows and macOS as well. It's fine to only implement it in Linux for today but let's have the names be just `debounce-duration` and `debounce-algorithm`.